<?xml version="1.0"?>
<launch>
    <!-- Simulator Bridge -->
    <include file="$(find styx)/launch/server.launch" />

    <!--DBW Node -->
    <include file="$(find twist_controller)/launch/dbw_sim.launch"/>

    <!--Waypoint Loader -->
    <include file="$(find waypoint_loader)/launch/waypoint_loader.launch"/>

    <!--Waypoint Follower Node -->
    <include file="$(find waypoint_follower)/launch/pure_pursuit.launch"/>

    <!--Waypoint Updater Node -->
    <include file="$(find waypoint_updater)/launch/waypoint_updater.launch"/>

    <!--Traffic Light Detector Node -->
    <include file="$(find tl_detector)/launch/tl_detector.launch"/>

    <!--Traffic Light Locations and Camera Config -->
    <param name="traffic_light_config" textfile="$(find tl_detector)/sim_traffic_light_config.yaml" />

    <param name="traffic_light_environment" value="Simulator" type="str" />
    <param name="traffic_light_model_name" value="model_sim.pb" type="str" />
    <param name="traffic_light_detection_threshhold" value="0.2" type="double" />
    <param name="traffic_light_max_distance" value="100.0" type="double" />
    <param name="yellow_max_braking_distance" value="40.0" type="double" />
    <param name="yellow_min_braking_distance" value="15.0" type="double" />


</launch>
